From f278a93493ac04ce04d48e8e5b5158caa0621ce3 Mon Sep 17 00:00:00 2001
From: Harald Sitter <sitter@kde.org>
Date: Thu, 7 Nov 2013 16:16:59 +0100
Subject: [PATCH 4/4] Ensure the PulseAudio envrionment is set up when running
 older backends

This ensures that pre-4.7 backends (expecting the environment to be
set up by libphonon) retain working volume control.

CCBUG: 327279
FIXED-IN: 4.7.1
---
 phonon/audiooutput.cpp | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/phonon/audiooutput.cpp b/phonon/audiooutput.cpp
index f40199e..dd33364 100644
--- a/phonon/audiooutput.cpp
+++ b/phonon/audiooutput.cpp
@@ -108,6 +108,8 @@ void AudioOutputPrivate::init(Phonon::Category c)
             AudioOutputInterface47 *iface = Iface<AudioOutputInterface47>::cast(this);
             if (iface)
                 iface->setStreamUuid(streamUuid);
+            else
+                pulse->setupStreamEnvironment(streamUuid);
         }
     }
 #endif
-- 
1.8.4.2

